### DISCUSSION TITLE
Does hypr have all these features?

### DISCUSSION THREAD

**Discussion by txtyash** [hyprwm/Hyprland#1467] [Q&A] [answered]
Does hyprland have these features:
1. Submaps
2. Scratchpads
3. Fake fullscreen
4. One sides border

**DETAILED:**
1. **Submaps:** Modal keybindings or nested bindings. Let's say I have named my workspaces alphabetically so for moving a window to workspace `A` then I might have a binding `mod+m` `a`, for moving to workspace `B`: `mod+m` `b`

1. **Scratchpads:** Bring up floating windows for quick note taking or any other application such as htop.

1. **Fake fullscreen:** Similar to DWM's fake fullscreen patch

1. **One sided border:** Window border only on one side:
![image](https://user-images.githubusercontent.com/85027668/216036771-2fa3fac7-d36c-4c9a-910a-0bbed8a31f93.png)

**Comment by JustSimplyKyle** [comment, marked as answer]
http://wiki.hyprland.org/Configuring/Binds/#submaps
http://wiki.hyprland.org/Configuring/Dispatchers/#special-workspace
http://wiki.hyprland.org/Configuring/Dispatchers/#list-of-dispatchers
![image](https://user-images.githubusercontent.com/68589851/216041488-f0918789-6d39-44c2-a62d-be3e9dc0f0fe.png)

**Reply by JustSimplyKyle** [reply]
> One sides border

It probably doesn't.

**Reply by JustSimplyKyle** [reply]
Also there is this search bar that *literally tells you where the page is if you search for submap/scratchpad/fake fullscreen*
![image](https://user-images.githubusercontent.com/68589851/216042373-737ee746-efc2-46a8-91e7-03f6aaedc8dd.png)

**Reply by vaxerski** [reply]
yep, all except the last. Thanks for being faster than me!

**Reply by txtyash** [reply]
And last thing, a universal light/dark theme toggle like gnome?

**Reply by JustSimplyKyle** [reply]
`gsettings set org.gnome.desktop.interface color-scheme "prefer-dark"` for gtk4
And nwg-look for gtk3
You can use qt5ct for well... Qt.
